.eslintignore

```diff
@@ -1 +1,2 @@
-helix-importer-ui
\ No newline at end of file
+helix-importer-ui
+scripts/video.js
```

Comment:
I think this can be removed again since you deleted `scripts/video.js`

Reply:
Good catch, right